lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -919,6 +919,92 @@ setReadOnly( Complex64Array.prototype, 'every', function every( predicate, thisA
 	return true;
 });
 
+/**
+* Returns the modified typed array filled with fill value.
+*
+* @name fill
+* @memberof Complex64Array.prototype
+* @type {Function}
+* @param {Complex64} value - fill value
+* @param {integer} [start=0] - starting index (inclusive)
+* @param {integer} [end] - ending index (exclusive)
+* @throws {TypeError} `this` must be a complex number array
+* @throws {TypeError} `value` must be a complex number
+* @throws {TypeError} `start` must be an integer
+* @throws {TypeError} `end` must be an integer
+*
+* @example
+* var realf = require( '@stdlib/complex/realf' );
+* var imagf = require( '@stdlib/complex/imagf' );
+*
+* var arr = new Complex64Array( 3 );
+*
+* arr.fill( new Complex64( 1.0, 1.0 ), 1 );
+*
+* var z = arr.get( 1 );
+* // returns <Complex64>
+*
+* var re = realf( z );
+* // returns 1.0
+*
+* var im = imagf( z );
+* // returns 1.0
```

Comment:
```suggestion
* var z = arr.get( 1 );
* // returns <Complex64>
*
* var re = realf( z );
* // returns 1.0
*
* var im = imagf( z );
* // returns 1.0
*
* z = arr.get( 1 );
* // returns <Complex64>
*
* re = realf( z );
* // returns 1.0
*
* im = imagf( z );
* // returns 1.0
```

It is fine to be a bit more verbose in examples, in order to more fully convey how a method/function works.